easypost/util.py

```diff
@@ -0,0 +1,36 @@
+from typing import List
+
+from easypost.easypost_object import EasyPostObject
+from easypost.error import Error
+
+
+class Util:
+    @staticmethod
+    def _get_lowest_object_rate(
+        easypost_object: EasyPostObject,
+        carriers: List[str] = None,
+        services: List[str] = None,
+        rates_key: str = "rates",
+    ):
```

Comment:
Add a return type hint, as well as a docstring (with parameters) for this method

Reply:
I've tried here and unfortunately to import `Rate` (the return type), we get a circular dependency error at runtime.

Reply:
You can use a [forward reference](https://peps.python.org/pep-0484/#forward-references), i.e. `) -> 'Rate':`

Reply:
That doesn't work either unfortunately because `"Rate"` not defined, linting complains without `Rate` being imported which is what leads to the circular import error.